components/hab/src/command/studio/enter.rs

```diff
@@ -17,6 +17,7 @@ use crate::{config,
 use habitat_core::AUTH_TOKEN_ENVVAR;
 
 pub const ARTIFACT_PATH_ENVVAR: &str = "ARTIFACT_PATH";
+pub const SSL_PATH_ENVVAR: &str = "CERT_PATH";
```

Comment:
Might as well call this `CERT_PATH_ENVVAR` now.

Reply:
Ok